llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -966,6 +995,39 @@ Error MCCASBuilder::buildFragments() {
       continue;
     }
 
+    if (Asm.getContext().getObjectFileInfo()->getDwarfLineSection() == &Sec) {
+      SmallVector<char> DebugLineContent;
+      for (const MCFragment &F : Sec) {
+        if (RelocLocation == Atom) {
+          auto Relocs = RelMap.find(&F);
+          if (Relocs != RelMap.end())
+            SectionRelocs.append(Relocs->second.begin(), Relocs->second.end());
```

Comment:
what happens if it's not in RelMap, is that handled below, or an error?

Reply:
I will suggest move this to the function where `RelMap` is built. You can skip building the map in DebugLine section and directly add to SectionRelocs because this section is encoded differently without considering Atoms.

Reply:
Yes I think this is a better approach and I will move the code there, thanks Steven!